Products/zms/__init__.py

```diff
@@ -77,7 +110,7 @@ def registerDirectory(_context, name, directory=None, recursive=False,
         filepath = path.join(_context.package.__path__[0], 'skins', str(name))
     else:
         # subdir = str(directory[len(_context.package.__path__[0]) + 1:])
-        subdir = str(name)
+        subdir = '/%s' % str(name)
```

Comment:
Check if / is needed fir UniBE
Check what happens if Monkey is applied twice - by zms-core and zms.addons